tinygrad/runtime/ops_cuda.py

```diff
@@ -44,20 +44,21 @@ class device:
     get_device = lambda: context.device # pylint: disable=unnecessary-lambda # noqa: E731
   import pycuda.driver
   pycuda.driver.Context = context
-  RawCUDABuffer = RawMallocBuffer
 else:
   import pycuda.autoprimaryctx # pylint: disable=unused-import # noqa: F401
-  import pycuda.driver as cuda # type: ignore
+  import pycuda.driver as pycuda_driver
   class CUDAAllocator(LRUAllocator):
     def __init__(self): super().__init__(self._get_cur_free_space(None))
-    def _do_alloc(self, size, dtype, device, **kwargs): return cuda.mem_alloc(size * dtype.itemsize) # type: ignore
+    def _do_alloc(self, size, dtype, device, **kwargs): return pycuda_driver.mem_alloc(size * dtype.itemsize)
     def _cached_bufkey(self, size, dtype, device): return (device, size*dtype.itemsize) # Buffers of the same length could be reused, no matter what dtype.
-    def _get_cur_free_space(self, device): return cuda.mem_get_info()[0] # type: ignore
+    def _get_cur_free_space(self, device): return pycuda_driver.mem_get_info()[0]
   CUDAAlloc = CUDAAllocator()
-  class RawCUDABuffer(RawBufferCopyInOut): # type: ignore
+  class RawCUDABufferCopyInOut(RawBufferCopyInOut):
     def __init__(self, size, dtype): super().__init__(size, dtype, allocator=CUDAAlloc)
-    def _copyin(self, x:np.ndarray, stream:Optional[cuda.Stream]=None): cuda.memcpy_htod_async(self._buf, x.ravel(), stream) # type: ignore
-    def _copyout(self, x:np.ndarray): cuda.memcpy_dtoh(x, self._buf) # type: ignore
+    def _copyin(self, x:np.ndarray, stream:Optional[pycuda_driver.Stream]=None): pycuda_driver.memcpy_htod_async(self._buf, x.ravel(), stream)
+    def _copyout(self, x:np.ndarray): pycuda_driver.memcpy_dtoh(x, self._buf)
+
+RawCUDABuffer = RawMallocBuffer if getenv("CUDACPU", 0) == 1 else RawCUDABufferCopyInOut
```

Comment:
need feedback here:
This change removes all the type: ignores from this file but it comes at the cost of calling the getenv for CUDACPU again. We could alternatively store the value at the start, do this call once and use that stored boolean value twice. Or, we could live with the `# type: ignore`s if that is fine for this case.

Reply:
two calls is fine, but `RawCUDABufferCopyInOut` is a bad name. think about what it is